app/scoring/page.tsx

```diff
@@ -5,13 +5,37 @@ import TeamList from '../components/teamList'
 import Team from '../models/Team'
 import { shouldBeScored } from '../utils/teamListUtils'
 
+interface Dictionary<T> {
+    [Key: string]: T;
+}
+
+function getKey(teamName: string): string {
```

Comment:
Move this function to the team object

Reply:
So I really like this suggestion, heck we work-shopped it together in-person, but after trying it out I realized that while this looks good on paper there are two uses of this function and the second one there is only a string and not a full team in order to call this method on. This leads to code that looks like this:
```
        const foundTeam = teamDictionary[(new Team({teamName: x})).getKey()]
```
Which doesn't quite feel right 🤔 

Also, trying to get too deep into this question seems like we are trying to solve #25 which I was really trying to avoid in this PR! 😅

Reply:
_update: so I learned that Javascript does have static functions so I tried out that and that wasn't half bad so I added those commits, would love to know what you think!_